src/net/tls.cc

```diff
@@ -387,6 +397,51 @@ class tls::certificate_credentials::impl: public gnutlsobj {
                 gnutls_certificate_set_x509_simple_pkcs12_mem(_creds, &w,
                         gnutls_x509_crt_fmt_t(fmt), password.c_str()));
     }
+    std::vector<cert_info> get_x509_info() const {
```

Comment:
These can't be used?

Reply:
Sorry, could you elaborate? Used directly by clients you mean?

Reply:
They look orphaned, in a .cc file without a header. Later I saw the user in the next patch.

Reply:
Gotcha, thx. Squashed the two commits as suggested.